insonmnia/hub/server.go

```diff
@@ -768,16 +775,18 @@ func (h *Hub) GetRegisteredWorkers(ctx context.Context, empty *pb.Empty) (*pb.Ge
 
 // RegisterWorker allows Worker with given ID to connect to the Hub
 func (h *Hub) RegisterWorker(ctx context.Context, request *pb.ID) (*pb.Empty, error) {
-	// todo: implement me
 	log.G(h.ctx).Info("handling RegisterWorker request", zap.String("id", request.GetId()))
-	return nil, ErrUnimplemented
+
+	h.acl.Insert(request.Id)
+	return &pb.Empty{}, nil
 }
 
 // DeregisterWorkers deny Worker with given ID to connect to the Hub
 func (h *Hub) DeregisterWorker(ctx context.Context, request *pb.ID) (*pb.Empty, error) {
-	// todo: implement me
 	log.G(h.ctx).Info("handling DeregisterWorker request", zap.String("id", request.GetId()))
-	return nil, ErrUnimplemented
+
+	h.acl.Remove(request.Id)
+	return &pb.Empty{}, nil
```

Comment:
Maybe it will be better to check if the key is present in the storage, then delete it. 
If there is no key, then fire an error.